include/librealsense2/hpp/rs_options.hpp

```diff
@@ -8,6 +8,25 @@
 
 namespace rs2
 {
+    template< class T >
+    class option_value_update_callback : public rs2_option_value_update_callback
```

Comment:
Doesn't need to be a template. See `core/frame-callback.h` for example (it used to be a template).

Reply:
Oh this is the rs2 interface... I thought it was librealsense. OK for now.